parser.ts

```diff
@@ -72,6 +77,14 @@ class Synchronize extends Error {
   }
 }
 
+@errorType('StopInstrs')
```

Comment:
I'm not thrilled about the addition of this exception type. Can I at least leave a comment about its motivation? Maybe I'll have an idea for getting rid of it at some point.